### ISSUE TITLE
Design Logos

### ISSUE THREAD

**Issue by IgorGee** [hunter695/icebreaker-app#13]
@MabelM1, would you mind pushing the finalized logos, banners, images, and anything else that'll be used in frontend implementation in the `src/assets` directory?

Also, within your git commit message (the long one, not the short one) just type the following:

`Fixes #<THIS_ISSUE_#>`

**Comment by MabelM1** [comment]
I see we only have a src folder so when I fork this into my own do you want me to create the folder assets in src and save the logo there? 
Also just to make sure- I will make these changes on a new branch then send you a pull request and you will merge it once approved. correct?

**Comment by IgorGee** [comment]
Yes, you'll have to make the folder.

That's not quite right. You make pull requests from forks of a repo.

There are essentially two workflows we can do:

* Fork -> clone fork -> make branch -> do stuff in that branch -> push to your fork -> make pull request from your fork

* (Only because you're a contributer) Pull directly from repo -> make branch -> do stuff in that branch -> merge branch into master -> push to repo

I suggest you do the first one.

**Comment by MabelM1** [comment]
I did the pull request. Do you want me to attach the front end mock up for your reference?

**Comment by IgorGee** [comment]
Yes, please upload the frontend mockup with the discussed changes as an album to imgur and post the link to the album on #2.

**Comment by MabelM1** [comment]
I tried to remove the background but since I added the cracking effect using vectors I can't get a transparent background. If we use google font effect ice we can get the same style.

**Comment by IgorGee** [comment]
Can you link me to that Google Font? I'm having trouble finding it.

**Comment by MabelM1** [comment]
font: roboto
https://fonts.google.com/specimen/Roboto

https://developers.google.com/fonts/docs/getting_started scroll towards the bottom of the page for the effects.
we can try the effect ice
